job/common/src/main/java/alluxio/job/workflow/WorkflowConfig.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.job.workflow;
+
+import alluxio.job.JobConfig;
+
+/**
+ * A workflow configuration.
+ */
+public interface WorkflowConfig extends JobConfig {
```

Comment:
Is this interface necessary (currently unused)?
Also, are workflows subclasses of jobs, I thought the relationship was Job = Task | Plan

Reply:
Job is everything because you need to be able to submit everything through the JobService.